code/datums/uplink_item.dm

```diff
@@ -577,11 +577,32 @@ var/list/uplink_items = list()
 
 /datum/uplink_item/ammo/pistol
 	name = "Magazine - 10mm"
-	desc = "An additional 8-round 10mm magazine for use in the syndicate pistol. These subsonic rounds are dirt cheap but are half as effective as .357 rounds."
+	desc = "An additional 8-round 10mm magazine for use in the syndicate pistol, loaded with standard rounds that are weak but inexpensive."
```

Comment:
Original description is a bit more informative to the purchaser, as it more concisely tells them just what, exactly, they're getting.